manifests/config.pp

```diff
@@ -39,6 +39,8 @@
       warn  => true,
     }
 
+    # Template uses:
+    # - $::openvpn::autostart_all
```

Comment:
I made the experience that comments like this outdate really fast and you cannot trust them... Could you remove this?